src/main/java/com/example/bookstore/service/ShoppingCartService.java

```diff
@@ -0,0 +1,19 @@
+package com.example.bookstore.service;
+
+import com.example.bookstore.dto.AddCartItemRequestDto;
+import com.example.bookstore.dto.ShoppingCartDto;
+import com.example.bookstore.dto.UpdateCartItemQuantityRequestDto;
+
+public interface ShoppingCartService {
+    ShoppingCartDto addBookToCart(AddCartItemRequestDto requestDto, Long userId);
+
+    ShoppingCartDto getUserCart(Long userId);
+
+    ShoppingCartDto updateQuantity(
+            Long userId,
+            UpdateCartItemQuantityRequestDto requestDto,
+            Long id
```

Comment:
In context of ShoppingCartService it's not obvious which ID it is. So be more specific
```suggestion
            Long cartItemId
```